### DISCUSSION TITLE
Required query parameter declaration issues (FastAPI v.0.100.0)

### DISCUSSION THREAD

**Discussion by paw-bor** [fastapi/fastapi#9862] [Questions] [answered]
### Privileged issue

- [X] I'm @tiangolo or he asked me directly to create an issue here.

### Issue Content

When I declare query parameter without Annotated and with Ellipsis, e.g.:
_item_name: str = ..._
Everythings works as it should, there are no errors and interactive API docs webpage contains an information that item_name is „*required”. But:

1.
When I declare query parameter with or without Annotated and without Ellipsis, (as it is described in Tutorial - User Guide), e.g.:
_item_name: str_
or 
_item_name: Annotated[str, Query(min_length=3)]_
I get an uvicorn process error:
    _item_name: str
    ^^^^^^^^^^^^^^
   SyntaxError: non-default argument follows default argument_

2.
When I declara query parameter with Annotated and with Ellipsis, e.g.:
_item_name: Annotated[str, Query(min_length=3)] = ..._
There are no errors but interactive API docs webpage DOESN’T contain any information that item_name is required.

3.
When I try to import Required from pydantic (as it is described in Tutorial - User Guide, eg at the chapter
https://fastapi.tiangolo.com/tutorial/query-params-str-validations/#use-pydantics-required-instead-of-ellipsis
e.g.

_from pydantic import Required
item_name: Annotated[str, Query(min_length=3)] = Required_
I got en error:
_pydantic.errors.PydanticImportError: `pydantic:Required` has been removed in V2._

**Comment by Kludex** [comment, marked as answer]
Please create a new discussion following the template provided: https://github.com/tiangolo/fastapi/discussions/new?category=questions

**Reply by ingarbi** [reply]
where is link to discussion?
